optashift-employee-rostering-gwtui/src/main/resources/org/optaplanner/openshift/employeerostering/gwtui/client/resources/i18n/OptaShiftUIConstants_en.properties

```diff
@@ -0,0 +1,28 @@
+ACTIONS=Actions
+ADD=Add
+CLICK_TO_START_SOLVING=Click the <i>Solve</i> button to start solving.
```

Comment:
Does wb also use capitals and underscore_case (as opposed to normal camelCase) for i18n keys?

Reply:
They use a modified form of camelCase, where the first letter is also capitalized (so "variable name" becomes "VariableName"). See here: https://github.com/kiegroup/kie-wb-common/blob/500f4ab3315f6d1a62f9d20ff0f72372361644b1/kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/status/empty/ContainerStatusEmptyView.html

Reply:
We should stick to wb conventions and follow that form of CamelCase.